src/bridge.c

```diff
@@ -348,6 +348,11 @@ int bridge__connect(struct mosquitto *context)
 	uint8_t notification_payload;
 	uint8_t qos;
 
+    //mosquitto_property receive_maximum
```

Comment:
Are these additional properties that you intend to also make configurable in the future?

Reply:
Right now, this is the only one we anticipate, but yes in theory more could be added in the future. We anticipate as mosquitto builds better support for MQTT 5 features that we wont have to add these patches in ourselves but rather they would be a part of the official moquitto application.